.travis.yml

```diff
@@ -17,6 +17,7 @@ matrix:
 
 install:
   - true
+  - script/test/cmd/fix_detached_head.sh
```

Comment:
Argh. I should of added a comment here.

Can you put:
```
// This is required because sometimes Travis ends up in a detached HEAD state after git clone
```

?